### PR TITLE
Edited meta.yaml for pycallgraph2.

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please ping team

File: recipes/pycallgraph2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pytest
```
Comment (ocefpaf) [inline recipes/pycallgraph2/meta.yaml:20]: This is probably not needed but if they added it in the setup.py and the package checks for it you probably cannot removed it. Do you want to try to remove it to see how it goes?

Comment (thewchan) [reply recipes/pycallgraph2/meta.yaml:20]: Sure, I'll do that.

Comment (thewchan) [reply recipes/pycallgraph2/meta.yaml]: @ocefpaf Looks like it didn't work? Should I revert it?

File: recipes/pycallgraph2/meta.yaml [hunk 2]
```diff
@@ -16,8 +16,6 @@ build:
 
 requirements:
   host:
-    - pip
```
Comment (ocefpaf) [inline recipes/pycallgraph2/meta.yaml]: Oh.  You removed pip too! It should be only pytest. You can try that in the feedstock later.

Comment (thewchan) [reply recipes/pycallgraph2/meta.yaml]: Sorry I misunderstood. Where can I try to change this?

Comment (ocefpaf) [reply recipes/pycallgraph2/meta.yaml]: There will be a new repository, where you'll have access so you can send updates to this packages, that we call feedstock. You can test it later there. Watch out for the GitHub invite.

**Pull request merged**